suite-native/app/eas.json

```diff
@@ -37,6 +37,10 @@
         },
         "adhoc": {
             "extends": "preview",
+            "env": {
+                "EXPO_PUBLIC_FF_IS_DEVICE_CONNECT_ENABLED": "true",
```

Comment:
I'm not sure how `extends` works exactly for object values but I can imagine this will override it and so EXPO_PUBLIC_ENVIRONMENT is missing here

Reply:
All the profiles already extend "base" and it seems to work correctly. 🤔